### DISCUSSION TITLE
Show pp in "Logged in" overlay

### DISCUSSION THREAD

**Discussion by yesseruser** [ppy/osu#27251] [Ideas] [answered]
The login overlay shows your current country and global raking. After the last update, when finishing a beatmap your current global ranking and your pp is shown next to your name for a while in the top bar.
It would only make sense to put your current pp into this overlay.

Another thing is, the rankings are kind of oddly aligned, so whilst implementing this, it could be aligned to the center or just distribute evenly on the given space.

![image](https://github.com/ppy/osu/assets/67008763/2c053514-6e18-4cfe-86b9-2bc7c034a63c)

**Comment by Flamiii** [comment, marked as answer]
https://github.com/ppy/osu/pull/27147 is relevant to this, although the design of how that might look is still WIP.

**Reply by yesseruser** [reply]
That looks like it would fix it, if it gets merged.
